msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -76,7 +75,52 @@ For example, you can customize app description by defining a new environment var
             "full": "Full description of tab0418"
         },
     }
-```
+   ```
+
+1. Since Teams Toolkit 5.10, if you would like to store the value of a field, such as a lengthy or multiline app description, in a separate text file, you could use the `file` function. For example, you can create description.txt file sotring the full description of your app in the parent folder of `manifest.json`. Then set the value of description.full as `$[file('description.txt')]` in `manifest.json`. Teams Toolkit will read the content from the text file and use it as full description when building app package zip. 
+
+   `description.txt`
+
+   ```text
+    This is the full description.
+   ```
+
+   `manifest.json`
+
+   ```json
+    {
+        "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+        "manifestVersion": "1.16",
+        "description": {
+            "short": "Short description of tab",
+            "full": "$[file('./description.txt')]"
+        },
+    }
+   ```
+
+   You could either put the file path in env/.env.xx and update the input parameter function of file function to placeholder in ${{xx_xx}} format.
```

Comment:
```suggestion
To add the file path, update it in either `env` or `.env.xx`. Then, modify the `file` function to serve as a placeholder in the `${{xx_xx}}` format for the input parameter function.
```

Reply:
This is another way user can set the parameter of file function. One way is to use it as "$[file('./description.txt')]" like what we introduced above. Another one, which we are introducing here, is to use environment variable. env/.env.xx is one file, not env or .env.xx. Could you help modify the string again?

Maybe something like "Or you could put the file path in env/.env.xx. Then, modify the parameter of `file` function to a placeholder in the `${{xx_xx}}` format."?